src/main/java/com/amazon/jenkins/ec2fleet/EC2FleetCloud.java

```diff
@@ -861,7 +861,7 @@ private void addNewSlave(final AmazonEC2 ec2, final Instance instance, FleetStat
         final EC2FleetAutoResubmitComputerLauncher computerLauncher = new EC2FleetAutoResubmitComputerLauncher(
                 computerConnector.launch(address, TaskListener.NULL));
         final Node.Mode nodeMode = restrictUsage ? Node.Mode.EXCLUSIVE : Node.Mode.NORMAL;
-        final EC2FleetNode node = new EC2FleetNode(instanceId, "Fleet slave for " + instanceId,
+        final EC2FleetNode node = new EC2FleetNode(instanceId, "Fleet node for " + instanceId,
```

Comment:
I think using agent instead of node is better as we refer to both node and computer as a slave / agent / worker.
Ref: https://www.jenkins.io/doc/book/managing/nodes/#creating-agents

Reply:
Makes sense, updated